Cargo.toml

```diff
@@ -94,4 +95,4 @@ required-features = ["std", "rand"]
 
 [[example]]
 name = "client-transports"
-required-features = ["net"]
\ No newline at end of file
+required-features = ["net"]
```

Comment:
Shouldn't this include the "unstable-client-transport" feature in the set of `required-features` ?